rocksdb/options/cache.nim

```diff
@@ -0,0 +1,12 @@
+import
+  ../lib/librocksdb
+
+type
+  CachePtr* = ptr rocksdb_cache_t
+
+  CacheRef* = ref object
+    cPtr*: CachePtr
+
+proc cacheCreateLRU*(size: int): CacheRef =
+  # TODO do you release it?
```

Comment:
It looks like you might need to call rocksdb_cache_destroy to release the handle.

Reply:
thanks, somehow I missed it ;)